core/types/outputpayload/transfercrosschainfee.go

```diff
@@ -0,0 +1,61 @@
+package outputpayload
+
+import (
+	"errors"
+	"io"
+
+	"github.com/elastos/Elastos.ELA/common"
+	"github.com/elastos/Elastos.ELA/crypto"
+)
+
+type CrossChainFee struct {
```

Comment:
AdditionalFee is better than CrossChainFee, and the filename should as the same as the class name.
Add comments to your code to explain why this output payload exists, and what's the parameters use for.
See `core/types/outputpayload/mapping.go` for example

Reply:
Already changed as required